USAGE.md

```diff
@@ -32,19 +50,18 @@ terminal.
 - On Linux, your distribution will provide a terminal; it can generally be opened with
 Ctrl+Shift+T or by searching for "Terminal" in your application menu.
 
-### On Docker and Non-Free Operating Systems ###
+### On OED and Non-UNIX Operating Systems ###
 
-Docker is well-supported on most free/libre operating systems, including GNU/Linux,
-FreeBSD, OpenBSD, and Plan9.
+Windows is not a supported platform. You can probably run OED there, but you will
+experience problems for the following reasons.
 
-Docker is not supported on Windows. Windows is incapable of supporting LXE containers,
-the technology on which Docker is based, and Microsoft
-[artifically limits](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/about/)
-the use of Docker for Windows, a virtualized technology.
+First, Docker is a Linux technology. It is based on Linux kernel features including namespaces
+and cgroups. On Mac OS, you can use [Docker for Mac](https://docs.docker.com/docker-for-mac/)
```

Comment:
If the section header is going to say "Non-Unix", then Mac OS should go somewhere else. Despite the fact that it's a different world from the other unices as far as docker is concerned, nobody is going to be looking for Mac instructions under the heading "Non-Unix".

Reply:
That's a good point, thanks! This language was here before this PR so I will move it in another PR.